lib/src/data/communication/api/api_key_local_repository.dart

```diff
@@ -54,7 +49,7 @@ class _ApiKeyRawRepository extends ApiKeyRepository {
     return rootBundle.loadStructuredData<ApiKey>(
       _Constants.newYorkTimesApiKeyRawLocation,
       (jsonStr) async {
-        final Map<String, dynamic> credentialsMap = json.decode(jsonStr);
+        final credentialsMap = json.decode(jsonStr) as Map<String, String>;
```

Comment:
dlaczego lepiej w ten sposób?

Reply:
Widocznie dartowy analyzer preferuje jawne castowanie ;)
Narusza to invalid_assignement:
https://dart.dev/tools/diagnostic-messages#invalid_assignment
Generalnie każdą z tych zasad można w analysis_options.yaml dać na ignore, w przypadku dynamic takie niejawne castowanie imo spoko, ale wyłączenie tej zasady nie zasygnalizuje jakiegoś ewentualnego głupiego przypisania :P